### DISCUSSION TITLE
How to build dbeaver on ARM pltaform machine?

### DISCUSSION THREAD

**Discussion by TuQinggang** [dbeaver/dbeaver#8525] [DBeaver Development] [answered]
#### System information: 
- Operating system (distribution) and version:deepin v20
- DBeaver version:7.0.3

#### Connection specification:
- Database name and version

#### Describe your question:
hello,now I should use dbeaver on a arm64 platform machine,How can I build it?

**Comment by serge-rider** [comment, marked as answer]
We don't support ARM platform officially. However there are some Eclipse builds for ARM. So potentially it is possible (open dbeaver in Eclipse and try to build it).

**Reply by iMonZ** [reply]
Hey,
since the new Apple M1 Macs are out for a while and the raspberry pi 4 gets more popular isn't it the time now to add official ARM support?

**Comment by serge-rider** [comment]
We added ARM support (GTK Linux) in version 21.

**Comment by rocky91-shi** [comment]
> #### System information:
> * Operating system (distribution) and version:deepin v20
> * DBeaver version:7.0.3
> 
> #### Connection specification:
> * Database name and version
> 
> #### Describe your question:
> hello,now I should use dbeaver on a arm64 platform machine,How can I build it?

Excuse me,

I also meet the same problem，What is the result of your arm dbeaver compilation?

**Reply by TuQinggang** [reply]
> > #### System information:
> > 
> > * Operating system (distribution) and version:deepin v20
> > * DBeaver version:7.0.3
> > 
> > #### Connection specification:
> > 
> > * Database name and version
> > 
> > #### Describe your question:
> > hello,now I should use dbeaver on a arm64 platform machine,How can I build it?
> 
> Excuse me,
> 
> I also meet the same problem，What is the result of your arm dbeaver compilation?

Sorry，I given up it finaly :worried:

**Reply by rocky91-shi** [reply]
Thank you very much for responding! I meet the same problem but for mips64el。

**Reply by rocky91-shi** [reply]
Is it possible using JDK-8 to transport dbeaer-21 which adds ARM support，or you have tried ?
